xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/internal/skin/EnvironmentSkin.java

```diff
@@ -52,7 +52,7 @@ public class EnvironmentSkin extends AbstractResourceSkin
      * @param logger a logger used to log warning in case of error when parsing a skin's syntax
      * @param environment the wiki environment, this is where this skin load its resources from
      * @param xcontextProvider a wiki context provide, used to give access to the context when resolving the skin's
-     *     rsources.
+     *            rsources.
```

Comment:
This was actually automatic formatting, did not really notice the change. Since there was no real modification to this file in the end, the best from the point of view of this PR might be to completely revert those changes.